pkg/ingester/transfer.go

```diff
@@ -624,176 +387,3 @@ func (i *Ingester) findTargetIngester(ctx context.Context) (*ring.IngesterDesc,
 
 	return &ingesters[0], nil
 }
-
-// unshippedBlocks returns a ulid list of blocks that haven't been shipped
```

Comment:
This function is useful for prometheus retention policy. We may want to revive it back soon.

Reply:
Yes, but could be slightly different (eg. moved it to a different struct). I would prefer to revive it when needed.